google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/sync/datacatalog_synchronizer.py

```diff
@@ -62,6 +62,14 @@ def run(self):
 
         return self.__task_id
 
+    def _create_consumer(self):
+        connection_config = {
+            'bootstrap.servers': self.__kafka_host,
+            'group.id': 'kafka2dc'
```

Comment:
@bonifacyj the `group.id` parameter should be in users' control, no?

they might use some pattern to name the Kafka consumer group in their production environment.  Because when  you connect to a topic with a consumer group, Kafka or Zookeper stores this information internally.

This should be at least a parameter in the constructor of the synchronizer class, and could have the default value `kafka2dc`.

Reply:
sure! I've made changes